### PR TITLE
Fixes issue #19254: With debug=2 going to Email Templates from Survey…

### PR DESCRIPTION
**Pull request by mfavetti**
… gives me 500 error

Fixes issue #19254: With debug=2 going to Email Templates from Survey gives me 500 error

### CODE REVIEW

**Review by Shnoulle — COMMENTED** [review]
More understandable code ?

File: application/controllers/admin/EmailTemplates.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function index($iSurveyId)
         foreach ($grplangs as $key => $grouplang) {
             $aData['bplangs'][$key] = $grouplang;
             $aData['attrib'][$key] = SurveyLanguageSetting::model()->find('surveyls_survey_id = :ssid AND surveyls_language = :ls', array(':ssid' => $iSurveyId, ':ls' => $grouplang));
```
[the remark this answers is not in the document]
Comment (mfavetti) [reply application/controllers/admin/EmailTemplates.php:68]: Sorry it took me so long to follow up.

I updated the PR to reflect your feedback. (You got me with that missing parenthesis on line 2 :P)

[the remark this answers is not in the document]
Comment (Shnoulle) [reply application/controllers/admin/EmailTemplates.php:68]: > I updated the PR to reflect your feedback. (You got me with that missing parenthesis on line 2 :P)

Yes, sorry … i just put sample of code without control. Just the idea.

**Review by Shnoulle — APPROVED** [review]
Thank you :+1:

**Pull request merged**